### PR TITLE
Actually use the given format to execute a query

### PR DESCRIPTION
**Pull request by mchacki**
This fixes a rolling upgrade issue 3.5.* -> devel

During upgrade the AQL should be performed in 3.5 format, this has not been forwarded correctly inside the DBServers.

Jenkins:
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/7282/

### CODE REVIEW

File: arangod/Aql/ExecutionEngine.cpp [hunk 1]
```diff
@@ -41,8 +41,8 @@
 #include "Basics/ScopeGuard.h"
 #include "Cluster/ServerState.h"
 #include "Futures/Utilities.h"
-#include "Logger/Logger.h"
 #include "Logger/LogMacros.h"
+#include "Logger/Logger.h"
```
Comment (hkernbach) [inline arangod/Aql/ExecutionEngine.cpp]: Can be removed . Guess it is not used anymore.

**Comment by hkernbach** [comment]
Upgade passed: 
```
2019-11-20T13:59:01Z [20398] INFO executing upgrade procedure: disabling server features
2019-11-20T13:59:01Z [20398] INFO [43396] {authentication} Jwt secret not specified, generating...
2019-11-20T13:59:02Z [20398] INFO [144fe] using storage engine 'rocksdb'
2019-11-20T13:59:02Z [20398] INFO [a1c60] {syscall} file-descriptors (nofiles) hard limit is unlimited, soft limit is 8192
2019-11-20T13:59:02Z [20398] INFO [3844e] {authentication} Authentication is turned off, authentication for unix sockets is turned on
2019-11-20T13:59:02Z [20398] INFO [0de5e] database upgrade passed
2019-11-20T13:59:02Z [20398] INFO [7da27] {startup} server will now shut down due to upgrade, database initialization or admin restoration.
```

Startup passed: 
```
2019-11-20T13:59:11Z [20434] INFO [e52b0] ArangoDB 3.6.0-devel [darwin] 64bit maintainer mode, using build heads/bug-fix/rolling-upgrade-aql-block-serialization-0-g5fd37e3c5b, VPack 0.1.33, RocksDB 6.2.0, ICU 58.1, V8 7.1.302.28, OpenSSL 1.1.1d  10 Sep 2019
2019-11-20T13:59:11Z [20434] WARNING [0458b] This is a maintainer version intended for debugging. DO NOT USE IN PRODUCTION!
2019-11-20T13:59:11Z [20434] WARNING [bd666] ==============================================================================
2019-11-20T13:59:11Z [20434] INFO [43396] {authentication} Jwt secret not specified, generating...
2019-11-20T13:59:11Z [20434] INFO [144fe] using storage engine 'rocksdb'
2019-11-20T13:59:11Z [20434] INFO [3bb7d] {cluster} Starting up with role PRIMARY
2019-11-20T13:59:11Z [20434] INFO [6ea38] using endpoint 'http+tcp://[::]:8540' for non-encrypted requests
2019-11-20T13:59:11Z [20434] INFO [a1c60] {syscall} file-descriptors (nofiles) hard limit is unlimited, soft limit is 8192
2019-11-20T13:59:11Z [20434] INFO [3844e] {authentication} Authentication is turned off, authentication for unix sockets is turned on
2019-11-20T13:59:11Z [20434] INFO [b6826] {cluster} Cluster feature is turned on. Agency version: {"server":"arango","license":"enterprise","version":"3.5.1"}, Agency endpoints: http+tcp://127.0.0.1:8531, http+tcp://127.0.0.1:8541, http+tcp://127.0.0.1:8551, server id: 'PRMR-a70cff21-6124-4910-b39d-c6dcac5f64b6', internal endpoint / address: tcp://localhost:8540', advertised endpoint: , role: 2
2019-11-20T13:59:11Z [20434] INFO [805b2] {cluster} using heartbeat interval value '1000 ms' from agency
2019-11-20T13:59:11Z [20434] INFO [cf3f4] ArangoDB (version 3.6.0-devel [darwin]) is ready for business. Have fun!
```

**Comment by hkernbach** [comment]
BUT, coordinator fails to start.

**Comment by mchacki** [comment]
Fixed the error reported by HKernbach

New jenkins
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/7300/

Please rereview

**Pull request merged**